src/utils/auth.tsx

```diff
@@ -137,7 +151,9 @@ export function useProvideAuth(authClient): AuthContext {
 
   return {
     isAuthenticated,
-    isAuthClientReady,
+    // TODO: Since we're also waiting on actor being ready too, this is not well
+    // named anymore
```

Comment:
Is it easy to resolve this todo before merging?

Reply:
Yea I came up with something more generic